### PR TITLE
Add title prop to the image component

### PR DESCRIPTION
**Pull request by Klynger**
#### What does this PR do? \*

As the title says

#### How to test it? \*

https://imagetitle--storecomponents.myvtex.com/admin/cms/site-editor/about-us

#### Related to / Depends on \*

https://github.com/vtex-apps/store-components/pull/699

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
```
Comment (iaronaraujo) [inline .vscode/settings.json:1]: Shouldn't this file be on .gitignore?

File: messages/en.json [hunk 2]
```diff
@@ -2,9 +2,11 @@
   "admin/editor.store-image.title": "Image",
   "admin/editor.store-image.description": "Show any image",
   "admin/editor.store-image.src.title": "Image",
-  "admin/editor.store-image.alt.title": "Alternative text",
+  "admin/editor.store-image.alt.title": "Alternative textttadsfiausfgiasdgfi",
```
Comment (iaronaraujo) [inline messages/en.json]:
```suggestion
  "admin/editor.store-image.alt.title": "Alternative text",
```

**Review by iaronaraujo — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (victorhmp) [reply .vscode/settings.json:1]: I would keep it out of the gitigore, since these could be useful.

File: messages/en.json [hunk 3]
```diff
@@ -2,9 +2,11 @@
...
   "admin/editor.store-image.blockClass.title": "CSS Block Class",
   "admin/editor.store-image.blockClass.description": "Adds an extra class name to ease styling",
+  "admin/editor.store-image.title.title": "Title of the image",
```
Comment (victorhmp) [inline messages/en.json]:
```suggestion
  "admin/editor.store-image.title.title": "Image title",
```

File: messages/en.json [hunk 4]
```diff
@@ -2,9 +2,11 @@
...
+  "admin/editor.store-image.title.description": "Title to be shown when you hover the image",
```
Comment (victorhmp) [inline messages/en.json]:
```suggestion
  "admin/editor.store-image.title.description": "Title to be shown on hover",
```

**Review by victorhmp — APPROVED** [review]

**Review by iaronaraujo — APPROVED** [review]

**Pull request merged**